src/gui/advancedsettings.cpp

```diff
@@ -640,6 +642,12 @@ void AdvancedSettings::loadAdvancedSettings()
     // Torrent added notifications
     m_checkBoxTorrentAddedNotifications.setChecked(mainWindow->isTorrentAddedNotificationsEnabled());
     addRow(TORRENT_ADDED_NOTIFICATIONS, tr("Display notifications for added torrents"), &m_checkBoxTorrentAddedNotifications);
+    // Notification timeout
+    m_spinBoxNotificationTimeout.setMinimum(1);
+    m_spinBoxNotificationTimeout.setMaximum(9999);
+    m_spinBoxNotificationTimeout.setValue(mainWindow->getNotificationTimeout());
+    m_spinBoxNotificationTimeout.setSuffix(tr(" s", "seconds"));
```

Comment:
IMO controlling at milliseconds would be better. Some users like to have finer control.